test/controllers/bigbluebutton_api_controller_test.rb

```diff
@@ -920,4 +919,66 @@ class BigBlueButtonApiControllerTest < ActionDispatch::IntegrationTest
     assert_not Recording.exists?(record_id: r1.record_id)
     assert_not Recording.exists?(record_id: r2.record_id)
   end
+
+  #RecordingDisabled
+ 
+  test 'getRecordings returns noRecordings if RECORDING_ENABLED flag is set to False' do
```

Comment:
@amalrails would not be this unless ENV['RECORDING_DISABLED'].eql?('TRUE')

All over the test

Reply:
Have updated it